meteor/client/ui/SegmentList/SegmentListHeader.tsx

```diff
@@ -121,31 +122,37 @@ export function SegmentListHeader({
 				{segment.name}
 			</h2>
 			<div className="segment-opl__counters">
-				<div
-					className={classNames('segment-opl__timeUntil', {
-						'segment-opl__timeUntil--time-of-day': useTimeOfDayCountdowns,
-					})}
-					onClick={onTimeUntilClick}
-				>
-					{playlist && parts && parts.length > 0 && showCountdownToSegment && (
-						<PartCountdown
-							partId={countdownToPartId}
-							hideOnZero={!useTimeOfDayCountdowns}
-							useWallClock={useTimeOfDayCountdowns}
-							playlist={playlist}
-							label={
-								useTimeOfDayCountdowns ? (
-									<span className="segment-timeline__timeUntil__label">{t('On Air At')}</span>
-								) : (
-									<span className="segment-timeline__timeUntil__label">{t('On Air In')}</span>
-								)
-							}
-						/>
-					)}
-					{studio.settings.preserveUnsyncedPlayingSegmentContents && segment.orphaned && (
-						<span className="segment-timeline__unsynced">{t('Unsynced')}</span>
-					)}
-				</div>
+				{segment.segmentTiming?.expectedStart || segment.segmentTiming?.expectedEnd ? (
+					<div className={classNames('segment-opl__expectedTime')} onClick={onTimeUntilClick}>
+						<BreakWallTime segment={segment} labelClassName="segment-timeline__expectedTime__label" />
+					</div>
+				) : (
+					<div
+						className={classNames('segment-opl__timeUntil', {
+							'segment-opl__timeUntil--time-of-day': useTimeOfDayCountdowns,
+						})}
+						onClick={onTimeUntilClick}
+					>
+						{playlist && parts && parts.length > 0 && showCountdownToSegment && (
+							<PartCountdown
+								partId={countdownToPartId}
+								hideOnZero={!useTimeOfDayCountdowns}
+								useWallClock={useTimeOfDayCountdowns}
+								playlist={playlist}
+								label={
+									useTimeOfDayCountdowns ? (
+										<span className="segment-timeline__timeUntil__label">{t('On Air At')}</span>
+									) : (
+										<span className="segment-timeline__timeUntil__label">{t('On Air In')}</span>
+									)
+								}
+							/>
+						)}
+						{studio.settings.preserveUnsyncedPlayingSegmentContents && segment.orphaned && (
+							<span className="segment-timeline__unsynced">{t('Unsynced')}</span>
+						)}
+					</div>
+				)}
```

Comment:
Looking at this repetitive code, I suspect we should refactor this to a separate component that could be shared across the headers, but I don't think the place to do this is in this PR